exercises/largest-series-product/largest-series-product.spec.js

```diff
@@ -27,6 +35,18 @@ describe('Series', () => {
     expect(new Series('99099').largestProduct(3)).toBe(0);
   });
 
+  xit('rejects invalid character in input', ()=> {
+    expect(()=> {
+      new Series('1234a5').largestProduct('2')
+    }).toThrow(new Error('Invalid input.'));
```

Comment:
The canonical specs talk about returning `-1` instead of throwing error. But, other existing specs were throwing, so I'd keep these throws for now.